plugins/chat/app/services/chat/lookup_channel_threads.rb

```diff
@@ -52,12 +52,14 @@ def can_view_channel(guardian:, channel:, **)
 
     def fetch_threads(guardian:, channel:, **)
       Chat::Thread
+        .strict_loading
         .includes(
           :channel,
-          :last_reply,
+          last_reply: [:uploads],
           original_message_user: :user_status,
-          original_message: :chat_webhook_event,
+          original_message: %i[chat_webhook_event chat_mentions chat_channel],
         )
+        .includes(original_message: { user: :user_status })
```

Comment:
Just to satisfy my curiosity 😉 I tried it in the console and looks like those can be merged:

```suggestion
          original_message: [:chat_webhook_event, :chat_mentions, :chat_channel, user: :user_status]
        )
```

Reply:
Oh really? I thought this syntax would world 🤔

Reply:
Well I just tried it too and it seems to work, interesting. Thanks :+1: